src/Module.h

```diff
@@ -15,6 +15,43 @@
 */
 class Module {
   public:
+    /*!
+     * Description of RF switch pin states for a single mode.
+     *
+     * See setRfSwitchTable() for details.
+     */
+    struct RfSwitchMode_t {
+      unsigned mode;
```

Comment:
Please use fully specified types (e.g. uint8_t).

Reply:
I think I used `unsigned` because this takes an enum, which I think resolves to `unsigned`. But you are right that just using a `uint8_t` is probably better (if that doesn't produce warnings, but I expect not), we won't have so many modes anyway.

Note that I didn't define the variable as a `Module::OpMode_t`, since it can also hold e.g. `STM32WLx::OpMode_t`.

Reply:
I've just seen too many bugs caused by assumed bit width of `int` (most recently in the on-board storage service of an unnamed spacecraft) to be comfortable with it.

If implicit conversion to e.g. uint8_t works, then I'm fine with it. If not, then explicit casting should be done.

Reply:
Resolved in a64ee65a11b104cec72ae28a512d875660c080a0.